client/src/modules/cohort/components/CohortHeader.jsx

```diff
@@ -128,22 +125,33 @@ class CohortHeader extends PureComponent {
     }
 
     if (nameToUpdate.length >= 1) {
-      updateCohort(cohortId, { name });
+      updateCohort(id, { name });
       this.setState({ isNameInputVisible: false });
     }
   };
 
-  handleDescriptionUpdate = cohortId => () => {
-    const { updateCohort, details } = this.props;
+  handleDescriptionUpdate = () => {
+    const {
+      updateCohort,
+      details,
+      match: {
+        params: { id }
+      }
+    } = this.props;
     const { description } = this.state;
     const { description: previousDescription } = details;
 
-    if (previousDescription === description) {
+    if (previousDescription.trim() === description.trim()) {
```

Comment:
is this necessary to trim previous values, which should be trimmed already?

Reply:
yes it's necessary